etc/coral-credits/app.py

```diff
@@ -0,0 +1,64 @@
+"""
+Application-specific settings.
+"""
+
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+# Database
+# https://docs.djangoproject.com/en/5.0/ref/settings/#databases
+
+DATABASES = {
+    "default": {
+        "ENGINE": "django.db.backends.sqlite3",
+        "NAME": "/data/db.sqlite3",
+    }
+}
```

Comment:
Maybe this should be in the settings in the chart values, so it is easier to change to point at a proper DB in future?

Reply:
made database configurable via values now